### DISCUSSION TITLE
What means CefErrorCode -30 ?

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#3784] [Q&A]
Hi! I get error when using LoadUrlAsyncResponse . HttpStatusCode is -1 and CefErrorCode is -30. However, havent found corresponding description here https://cefsharp.github.io/api/63.0.0/html/T_CefSharp_CefErrorCode.htm . What it means?

**Comment by campersau** [comment]
According to https://chromium.googlesource.com/chromium/src/+/refs/heads/main/net/base/net_error_list.h#124 `-30` is `BLOCKED_BY_CSP`.
Looks like `CefErrorCode` would need to be updated.

**Reply by assopri** [reply]
Hi! Thank you! Can I somehow turn -30 or any other value of CefErrorCode (1) to the constant name (2) OR to its description (3) to give user more informative log, than just -30 ?
![image](https://user-images.githubusercontent.com/73235464/131786097-10efe7cb-5d8c-4775-8ae1-28dec0ec6c24.png)

**Reply by campersau** [reply]
Normally you can call `ToString()` to get the constant name (3) but since `-30` isn't currently defined in CefSharp you would need to make a special case for it yourself.

https://github.com/cefsharp/CefSharp/pull/3785 will [add this](https://github.com/cefsharp/CefSharp/pull/3785/files#diff-c22d802139a41b18f839ed894658590520416565d304bc8a7a5396c50a840ed8R188-R191) (and other) missing enum values, so after that you don't need to special case it anymore but it will take some time until it is released.

**Reply by assopri** [reply]
Thank you!

**Reply by amaitland** [reply]
`93.1.110-pre` release is available including updated error codes, see https://github.com/cefsharp/CefSharp/issues/3780#issuecomment-913087509 for release details.
